readme.md

```diff
@@ -15,6 +14,8 @@ About Laravel Zero
 Laravel Zero was created by [Nuno Maduro](https://github.com/nunomaduro) and it is maintained by [Nuno Maduro](https://github.com/nunomaduro) & [Harish Toshniwal](https://github.com/introwit). Laravel Zero is a micro-framework that provides an elegant starting point for your next console application.
 **Unofficial** and customized version of Laravel optimized for building console/shell/command-line applications.
 
+Laravel Zero logo was made by [Caneco](https://github.com/caneco), inspired by the Laravel logo family designed by [@steveschoger](https://twitter.com/steveschoger).
```

Comment:
You can remove this. I will create an CREDITS.MD to all maintainers, contributors, etc. Then I will add the credits for this logo.